llvm/test/CodeGen/CHERI-Generic/Inputs/frameindex-arith.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: llc @PURECAP_HARDFLOAT_ARGS@ %s -o - | FileCheck %s
+
+; Check that we can fold the GEP (PTRADD) into the FrameIndex calculation
+; rather than emitting two instructions.
+
+; Contains an explicit @llvm.cheri.cap.bounds.set so CheriBoundAllocas sees the
+; use as safe and doesn't interfere by inserting bounds on the FrameIndex
```

Comment:
Could also disable the pass using the command line flag?

Reply:
I did it this way because Morello still has its own so the flag won't work there (though their pass isn't smart enough to omit the SCBNDS...)

Reply:
Ah yes that makes sense.